flight/targets/flyingf3/fw/pios_board.c

```diff
@@ -710,6 +710,16 @@ void PIOS_Board_Init(void) {
 	if (PIOS_BMP085_Test() != 0)
 		panic(5);
 #endif /* PIOS_INCLUDE_BMP085 && PIOS_INCLUDE_I2C */
+#if defined(PIOS_INCLUDE_ADC)
```

Comment:
1. That's up to you, you can change it if you want. :-)
2. No, anything you push to this branch(findus4fun:next) goes into this PR. ;-) It's a good idea to create a separate branch for each PR you do in the future.